_includes/home-page.html

```diff
@@ -65,7 +65,7 @@
     }
 
     ::-webkit-scrollbar-thumb {
-      background-color: var(--color-primary-dark);
+      background-color: #00B39F;
```

Comment:
@Faizan711, please use theme ref color, I see this color is in our brand guide, if it's not in them you can add it and then use it.